### ISSUE TITLE
在 app.tsx 中不生效

### ISSUE THREAD

**Issue by yimiIO** [MinaAndLuoXia/Taro-i18n#3]
应该是初始化还没有完成，  另外是否可能支持 RN?

**Comment by MinaAndLuoXia** [comment]
> 应该是初始化还没有完成，另外是否可能支持RN？

是的,如果初始化执行过晚或异步执行,在初始化之前使用会无效,需提升初始化执行顺序,至于RN我本地没有RN测试环境,你可以试一下可不可以使用,可能Taro兼容性做的比较好,可以正常使用也说不定=.=

**Comment by yimiIO** [comment]
RN不支持，我看看看能不能解决。 app. tsx内得国际化官方fix一下吧



---原始邮件---
发件人: "MinaAndLuoXia"<notifications@github.com&gt;
发送时间: 2020年6月6日(周六) 下午5:38
收件人: "MinaAndLuoXia/Taro-i18n"<Taro-i18n@noreply.github.com&gt;;
抄送: "Author"<author@noreply.github.com&gt;;"yimi.io"<zpy@98ki.com&gt;;
主题: Re: [MinaAndLuoXia/Taro-i18n] 在 app.tsx 中不生效 (#3)




  
应该是初始化还没有完成，另外是否可能支持RN？
  
是的,如果初始化执行过晚或异步执行,在初始化之前使用会无效,需提升初始化执行顺序,至于RN我本地没有RN测试环境,你可以试一下可不可以使用,可能Taro兼容性做的比较好,可以正常使用也说不定=.=
